bin/utils/constants.js

```diff
@@ -0,0 +1,37 @@
+const NON_COMPILATION_ARGS = [
+	"init",
+	"migrate",
+	"add",
+	"remove",
+	"serve",
+	"generate-loader",
+	"generate-plugin",
+	"info"
+];
+
+const CONFIG_GROUP = "Config options:";
+const BASIC_GROUP = "Basic options:";
+const MODULE_GROUP = "Module options:";
+const OUTPUT_GROUP = "Output options:";
+const ADVANCED_GROUP = "Advanced options:";
+const RESOLVE_GROUP = "Resolving options:";
+const OPTIMIZE_GROUP = "Optimizing options:";
+const DISPLAY_GROUP = "Stats options:";
+const GROUPS = {
+	CONFIG_GROUP,
```

Comment:
Wouldn't be more compact to initialize the constants inside the JSON? :)

```js
const GROUPS = {
	CONFIG_GROUP: "Config options:"
```

Reply:
What do you mean by "compact"?